server/server.go

```diff
@@ -732,6 +732,13 @@ func (s *Server) KillAllConnections() {
 		}
 		killConn(conn)
 	}
+
+	if s.dom != nil {
+		sysProcTracker := s.dom.SysProcTracker()
+		for connID := range sysProcTracker.GetSysProcessList() {
+			sysProcTracker.KillSysProcess(connID)
+		}
+	}
```

Comment:
We also need to kill auto analyze here because
https://github.com/pingcap/tidb/blob/ebdc78463a6ff52262183b67aec6c905e921be9a/domain/domain.go#L1780-L1783